src/views/Authorizations/ViewAuthorization/ViewAuthorization.js

```diff
@@ -0,0 +1,182 @@
+        if (error.response) {
+            // The request was made and the server responded with a status code
+            // that falls out of the range of 2xx
+            message = message.concat(`Mensaje del servidor: ${error.response.statusText}`);
+        } else if (error.request) {
+            // The request was made but no response was received
+            // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
+            // http.ClientRequest in node.js
+            message = message.concat('No pudo establecerse comunicación con el servidor.');
+        } else {
+            // Something happened in setting up the request that triggered an Error
+            message = message.concat('No pudo realizarse el pedido al servidor');
+        }
+        this.setState({ alertColor: 'danger', isAlertVisible: true, alertMessage: message });
+    };
+
+    render() {
+        return (
+            <div>
+                <Alert color={this.state.alertColor} isOpen={this.state.isAlertVisible} name="isAlertVisible" toggle={() => this.handleAlertDismiss("isAlertVisible")}>
+                    {this.state.alertMessage}
+                </Alert>
+                <Card>
+                    <CardHeader>
+                        <strong>Autorización</strong>
+                    </CardHeader>
+                    <CardBody>
+                        <Form id="authorization-form" className="form-horizontal" onSubmit={this.handleSubmit}>
+                            <FormGroup row>
+                                <Col md="3">
+                                    <Label htmlFor="text-input">Titulo</Label>
+                                </Col>
+                                <Col xs="12" md="9">
+                                    <Input type="text" id="title-input" name="title" value={this.state.authorization.title} onChange={this.handleChange} disabled={true} />
+                                </Col>
+                            </FormGroup>
+                            <FormGroup row>
+                                <Col md="3">
+                                    <Label htmlFor="status-input">Estado</Label>
+                                </Col>
+                                <Col xs="12" md="9">
+                                    <Input type="text" id="status-input" name="status" disabled={true} value={this.state.authorization.status} onChange={this.handleChange} />
+                                </Col>
+                            </FormGroup>
+                            <FormGroup row>
+                                <Col md="3">
+                                    <Label htmlFor="status-input">Tipo de procedimiento</Label>
+                                </Col>
+                                <Col xs="12" md="9">
+                                    <Input type="text" id="status-input" name="procedure" disabled={true} value={this.state.authorization.type} onChange={this.handleChange} />
```

Comment:
`Warning: A component is changing a controlled input of type text to be uncontrolled.`
No entiendo por qué pasa, no veo el problema en lo nuestro.